gene_finder.py

```diff
@@ -2,7 +2,7 @@
 """
 YOUR HEADER COMMENT HERE
 
-@author: YOUR NAME HERE
+@author:
```

Comment:
Please write your name. It's like writing your name for any other quizzes or projects.